test/views/partials/statuses/AppealStatusViewSpec.scala

```diff
@@ -83,7 +83,7 @@ class AppealStatusViewSpec extends ViewSpec {
 
       val doc = view(appeal_status(c, "id"))
 
-      doc.text() shouldBe "Completed"
+      doc.text() shouldBe "COMPLETED"
     }
 
     //REVIEW
```

Comment:
remove this?

Reply:
review is type of appeal its not todo in code, just to group tests by type